python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py

```diff
@@ -94,8 +95,11 @@ def dirac_job(load_config):
     """
     api_script = dedent(api_script)
 
+    cred_req = DiracProxy(group='lhcb_user')
```

Comment:
Do I need to change the user? Which group is the certificate on the Jenkins server a member of?